catroid/src/main/java/org/catrobat/catroid/content/bricks/CloneBrick.java

```diff
@@ -70,16 +70,16 @@ public View getView(Context context) {
 	}
 
 	@Override
-	public void onNewOptionSelected() {
+	public void onNewOptionSelected(Integer spinnerId) {
 	}
 
 	@Override
-	public void onStringOptionSelected(String string) {
+	public void onStringOptionSelected(Integer spinnerId, String string) {
```

Comment:
Clonebrick as well

Reply:
the same reason as above applies to every similar codechange in different files that added the id as a parameter